src/blueprints/organization/templates/organization.html

```diff
@@ -1,14 +1,29 @@
 {% extends 'base.html' %}
 
 {% block header %}
-  <h1>{% block title %}Hello World{% endblock %}</h1>
+  <h1>{% block title %}[Organization] View{% endblock %}</h1>
   <link rel="stylesheet" href="{{ url_for('bp_organization.static', filename='css/styles.css') }}">
-  <script src="{{ url_for('bp_organization.static', filename='js/charts.js') }}"></script>{% endblock %}
+  <script src="{{ url_for('bp_organization.static', filename='js/charts.js') }}" defer></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.js"></script>{% endblock %}
```

Comment:
Script tags should be placed at the bottom of html

Reply:
Moved to bottom as requested